packages/performance/src/types.ts

```diff
@@ -5,6 +5,7 @@ export enum EventType {
   TimeToLargestContentfulPaint = 'ttlcp',
   DomContentLoaded = 'dcl',
   FirstInputDelay = 'fid',
+  CumulativeLayoutShift = 'cls',
```

Comment:
(passing by as I am interested by that PR)

It should also be added to the [README list of lifecycle events](https://github.com/Shopify/quilt/tree/main/packages/performance#events)

Reply:
Thanks! Added here: 3f30837194f1cdaac00997629a93680f8b5fd1f6